cypress/e2e/models/administration/jira/jira.ts

```diff
@@ -0,0 +1,237 @@
+    }
+
+    /**
+     * This method is selecting credentials for Jira connection
+     */
+    private selectCredentials(): void {
+        click(selectCredentialToggle);
+        clickByText(button, this.credential.name);
+    }
+
+    /**
+     * This method is enabling/disabling insecure connection according to value in object
+     */
+    private configureInsecure(): void {
+        if (this.isInsecure) {
+            enableSwitch("#insecure-switch");
+        } else {
+            disableSwitch("#insecure-switch");
+        }
+    }
+
+    /**
+     * This method creates new Jira connection according to object values
+     *
+     * @param toBeCanceled is responsible for canceling creation instead of submitting if set to `true`
+     */
+    public create(toBeCanceled = false): void {
+        Jira.openList();
+        this.fillName();
+        this.fillUrl();
+        this.selectType();
+        this.selectCredentials();
+        this.configureInsecure();
+        if (!toBeCanceled) {
+            submitForm();
+            this.validateState();
+        } else {
+            cancelForm();
+            notExists(this.name);
+        }
+    }
+
+    /**
+     * This method edits existing Jira connection according to object values
+     *
+     * @param jiraConnectionData brings new values that should be applied to instance.
+     * @param toBeCanceled is responsible for canceling editing instead of submitting if set to `true`
+     */
+    public edit(jiraConnectionData: JiraConnectionData, toBeCanceled = false): void {
+        Jira.openList();
+        performRowAction(this.name, editAction);
+        const oldValues = this.storeOldValues();
+        this.init(jiraConnectionData);
+        this.fillName();
+        this.fillUrl();
+        this.selectType();
+        this.selectCredentials();
+        this.configureInsecure();
+        if (!toBeCanceled) {
+            // Edit action is confirmed, submitting form
+            submitForm();
+        } else {
+            // Edit action was canceled
+            this.init(oldValues);
+            cancelForm();
+        }
+        this.validateState();
+        exists(this.name);
+    }
+
+    /**
+     * This method deletes Jira connection
+     *
+     * @param toBeCanceled is responsible for canceling deletion instead of submitting if set to `true`
+     */
+    public delete(toBeCanceled = false): void {
+        Jira.openList();
+        performRowAction(this.name, deleteAction);
+        if (toBeCanceled) {
+            click(cancelButton);
+            exists(this.name);
+        } else {
+            click(confirmButton);
+            notExists(this.name);
+        }
+    }
+
+    /**
+     * This method validates all fields values of Jira connection after creation
+     */
+    private validateState(): void {
+        cy.get(tdTag, { timeout: 120 * SEC })
+            .contains(this.name, { timeout: 120 * SEC })
+            // .closest(tdTag)
```

Comment:
Here unused tdTag can be removed from the code

Reply:
removed, thank you for noticing this